packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -54,4 +57,36 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         // Add the path and icon div before the `inputField`.
         this.contentNode.insertBefore(element, this.inputField);
     }
+
+    /**
+     * Append the human-readable absolute root `path` to the dialog.
+     */
+    protected appendRootName(): void {
+        if (this.workspaceService && this.workspaceService.isMultiRootWorkspaceOpened) {
+            const roots = this.workspaceService.tryGetRoots();
+            const RootsWithSameName = roots.filter(singleRoot => {
```

Comment:
Please use `camelCasing` for local-variables.
The following goes against our [coding-guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#camelcase-var).